.github/workflows/recipe_test.yaml

```diff
@@ -41,7 +41,7 @@ jobs:
         run: |
           python -m pip install torch torchvision torchao
           python -m pip install -e ".[dev]"
-          python -m pip install lm-eval==0.4.*
+          python -m pip install git+https://github.com/EleutherAI/lm-evaluation-harness.git@fb963f0f0a5b28b69763590bb59676072cf43a01
```

Comment:
Do we plan to add something on this to our readme or something?

Reply:
README would be ideal

Reply:
U gonna add?

Reply:
no u

Reply:
Sorry, why are we pinning to a commit here?